crates/utils/src/signing.rs

```diff
@@ -9,6 +9,18 @@ use ethereum_consensus::{
     Error, Fork,
 };
 
+pub fn verify_constraints_signature<T: Merkleized>(
+    message: &mut T,
+    signature: &BlsSignature,
+    public_key: &BlsPublicKey,
+    context: &Context,
```

Comment:
As we said `context` can be taken from the `chain_info` that is available in the api.